File: GrowthWare_VB/GrowthWareFrameworkModel/Profiles/MFunctionProfileCollection.vb
Imports System.Runtime.Serialization
Imports GrowthWare.Framework.Model.Profiles.Base

Namespace Profiles
	<Serializable(), CLSCompliant(True)> _
	Public NotInheritable Class MFunctionProfileCollection
		Inherits MProfileCollection

		Public Sub New()

		End Sub

		Private Sub New(ByVal info As SerializationInfo, ByVal context As StreamingContext)
			MyBase.New(info, context)
		End Sub
	End Class
End Namespace

File: GrowthWare_CS/GrowthWareFramework/ModelObjects/MFunctionProfileCollection.cs
using System;
using System.Runtime.Serialization;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public sealed class MFunctionProfileCollection : MProfileCollection
	{
		MFunctionProfileCollection() : base()
		{
			// do nothing
		}
		MFunctionProfileCollection(SerializationInfo info, StreamingContext context) : base(info, context)
		{
			// do nothing
		}
	}
}
